.gitlab-ci.yml

```diff
@@ -167,8 +167,8 @@ variables:
   DATADOG_AGENT_BTF_GEN_BUILDIMAGES: v21999990-f991d55
   # To use images from test-infra-definitions dev branches, set the SUFFIX variable to -dev
   # and check the job creating the image to make sure you have the right SHA prefix
-  TEST_INFRA_DEFINITIONS_BUILDIMAGES_SUFFIX: ""
-  TEST_INFRA_DEFINITIONS_BUILDIMAGES: 8d38c2a2a794
+  TEST_INFRA_DEFINITIONS_BUILDIMAGES_SUFFIX: "-dev"
+  TEST_INFRA_DEFINITIONS_BUILDIMAGES: af0eb875
```

Comment:
```suggestion
  TEST_INFRA_DEFINITIONS_BUILDIMAGES: af0eb8752069
```

Reply:
The `dev-` version uses `CI_COMMIT_SHORT_SHA` while prod one uses `CI_COMMIT_SHA:0:12`. As soon as `TEST_INFRA_DEFINITIONS_BUILDIMAGES_SUFFIX` is updated to `""` this would change too. @usamasaqib you could add a similar action to protect us from merging using dev versions of the build images,  likely done in `datadog-agent` at https://github.com/DataDog/datadog-agent/pull/20194

Reply:
Thank you! I will take a look at this.